cmd/mdatagen/statusdata.go

```diff
@@ -12,13 +12,14 @@ import (
 // - The distribution must be open source.
 // - The link must point to a publicly accessible repository.
 var distros = map[string]string{
-	"core":     "https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol",
-	"contrib":  "https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib",
-	"aws":      "https://github.com/aws-observability/aws-otel-collector",
-	"observiq": "https://github.com/observIQ/observiq-otel-collector",
-	"splunk":   "https://github.com/signalfx/splunk-otel-collector",
-	"sumo":     "https://github.com/SumoLogic/sumologic-otel-collector",
-	"redhat":   "https://github.com/os-observability/redhat-opentelemetry-collector",
+	"core":          "https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol",
+	"contrib":       "https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib",
+	"aws":           "https://github.com/aws-observability/aws-otel-collector",
+	"grafana-agent": "https://github.com/grafana/agent",
```

Comment:
Is there another distribution from Grafana? If not, maybe just `grafana` given that we keep other distro names short?

Reply:
IMO This makes sense given what you are doing with the others. Normally we say grafana agent due to how common the term agent is.